### PR TITLE
DOC: Correct spacing around colon in docstrings

### PR DESCRIPTION
**Pull request by bashtage**
Ensure that param : type is correctly spaced

- [X] code/documentation is well formatted.  
- [X] properly formatted commit message. See 
      [NumPy's guide](https://docs.scipy.org/doc/numpy-1.15.1/dev/gitwash/development_workflow.html#writing-the-commit-message). 

<details>

**Notes**:

* It is essential that you add a test when making code changes. Tests are not 
  needed for doc changes.
* When adding a new function, test values should usually be verified in another package (e.g., R/SAS/Stata).
* When fixing a bug, you must add a test that would produce the bug in master and
  then show that it is fixed with the new code.
* New code additions must be well formatted. Changes should pass flake8. If on Linux or OSX, you can
  verify you changes are well formatted by running 
  ```
  git diff upstream/master -u -- "*.py" | flake8 --diff --isolated
  ```
  assuming `flake8` is installed. This command is also available on Windows 
  using the Windows System for Linux once `flake8` is installed in the 
  local Linux environment. While passing this test is not required, it is good practice and it help 
  improve code quality in `statsmodels`.
* Docstring additions must render correctly, including escapes and LaTeX.

</details>

### CODE REVIEW

File: statsmodels/stats/outliers_influence.py [hunk 1]
```diff
@@ -736,7 +736,8 @@ def dffits_internal(self):
 
     @cache_readonly
     def dffits(self):
-        """dffits measure for influence of an observation
+        """
+        dffits measure for influence of an observation
```
Comment (josef-pkt) [inline statsmodels/stats/outliers_influence.py:740]: I'm keeping those on the first line if it is short enough

File: statsmodels/stats/rates.py [hunk 2]
```diff
@@ -30,17 +30,17 @@ def test_poisson_2indep(count1, exposure1, count2, exposure2, ratio_null=1,
 
     Parameters
     ----------
-    count1: int
+    count1 : int
```
Comment (josef-pkt) [inline statsmodels/stats/rates.py:33]: those changes will cause merge conflicts with the docs that I'm working on right now
proportion, rates, oneway, meta_analysis, ...

Comment (josef-pkt) [reply statsmodels/stats/rates.py:33]: might cause*

Comment (bashtage) [reply statsmodels/stats/rates.py:33]: I checked, it is a very simple rebase that is already needed.

**Comment by josef-pkt** [comment]
what happened with this commit https://github.com/statsmodels/statsmodels/commit/d666e01acad0a901469b0674dfe289b7efb730b2

I get merge conflicts all over in parts that I just edited in my PR
e.g. https://github.com/statsmodels/statsmodels/commit/d666e01acad0a901469b0674dfe289b7efb730b2#diff-36a55e208f6d1db357321ee87ef3ab77R1428

oneway looks a mess

I don't need to fix my docstrings if you edit them at the same time

**Comment by bashtage** [comment]
I rebased your PR on top of this PR and there was an easy to resolve conflict in each file where you can just take your version in a single block over master, if you wish.

**Pull request merged**